src/main/java/com/saralein/server/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.saralein.server;
+
+import java.util.HashMap;
+
+public final class Constants {
+    private Constants() {}
+
+    public static final String CRLF = "\r\n";
+
+    public static final HashMap<Integer, String> STATUS_CODES = new HashMap<Integer, String>() {{
+        put(200, "200 OK");
+        put(404, "404 Not Found");
+    }};
```

Comment:
I think you can still call `STATUS_CODES.put(1000000, "100000000 ERROR")` and modify that list

Reply:
(I guess that's not really the real concern, the real concern is overwriting a status code)